src/PlatformSwitch/PlatformSwitch.tsx

```diff
@@ -79,6 +80,7 @@ export const PlatformSwitch: FC<PlatformSwitchProps> = ({
   platformsProps,
 }) => {
   /** Helper inner component: generates buttons from icons while capturing parent props */
+  const { isMobile } = useMobileView();
```

Comment:
Move this line up, so the comment that refers to the component below stays relevant.